GliaWidgets/Public/Glia/Glia.Deprecated.swift

```diff
@@ -185,6 +185,47 @@ extension Glia {
         )
     }
 
+    /// Deprectated, use ``Glia.startEngagement(engagementKind:in queueIds:theme:features:sceneProvider:)`` instead. 
```

Comment:
would it make sense for you to move this deprecation to the end of this file, so we will have all deprecation sorted from top to bottom by date and it will be clear vision who is the next candidate for marking `unavailable`?

Reply:
fixed [dab4c0d](https://github.com/salemove/ios-sdk-widgets/pull/835/commits/dab4c0da8a4e58738a342e0f67b2c77b8b3db6a6)